features/TESTS/filesystem/basic/basic.cpp

```diff
@@ -303,7 +301,7 @@ static control_t fsfat_basic_test_02()
     static const char hello[] = "Hello, world.\n";
     static const char replace[] = "Hewwo, world.\n";
     static const size_t replace_from = 2, replace_to = 4;
-    char* filename = (char*) sd_file_path; //todo: remove
+    char* filename = (char*) sd_file_path;
```

Comment:
small nit: should this actually be `const char *filename = sd_file_path`? not sure filename should/can be mutable.